languages/templates/src/main/webapp/i18n/ru/_global.json

```diff
@@ -94,25 +94,25 @@
             "title": "Подтвердите операцию удаления"
         },
         "validation": {
-            "required": "This field is required.",
-            "minlength": "This field is required to be at least {{min}} characters.",
-            "maxlength": "This field cannot be longer than {{max}} characters.",
-            "min": "This field should be more than {{min}}.",
-            "max": "This field cannot be more than {{max}}.",
-            "minbytes": "This field should be more than {{min}} bytes.",
-            "maxbytes": "This field cannot be more than {{max}} bytes.",
-            "pattern": "This field should follow pattern {{pattern}}.",
-            "number": "This field should be a number.",
-            "datetimelocal": "This field should be a date and time."
+            "required": "Это поле обязательно к заполнению.",
+            "minlength": "Это поле является обязательным, по крайней мере {{min}} символов.",
+            "maxlength": "Это поле не может быть длинее, чем {{max}} символов.",
+            "min": "Это поле должно быть больше, чем {{min}}.",
+            "max": "Это поле не может быть больше, чем {{max}}.",
+            "minbytes": "Это поле должно быть больше, чем {{min}} байтов.",
+            "maxbytes": "Это поле не может быть больше, чем {{max}} байтов.",
+            "pattern": "Это поле должно следовать модели {{pattern}}.",
+            "number": "Это поле должно быть число.",
+            "datetimelocal": "Это поле должно быть дата и время."
         }
     },
     "error": {
-        "server.not.reachable": "Server not reachable",
-        "NotNull": "Field {{fieldName}} cannot be empty!",
-        "Size": "Field {{fieldName}} does not meet min/max size requirements!",
-        "userexists": "Login name already used!",
-        "emailexists": "E-mail is already in use!",
-        "idexists": "A new {{entityName}} cannot already have an ID"
+        "serverNotReachable": "Сервер не доступен",
```

Comment:
why did you change the key from `server.not.reachable` to `serverNotReachable` ?